setup_all.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+source config_vars.sh
+
+set -e
+
+# Start MySQL server
+./run_mysql.sh
+
+# Build and run the API
+docker build -t zeeguu-api-core -f docker-files/zeeguu-api-core/Dockerfile .
+docker run --net=host -d \
+    -e MICROSOFT_TRANSLATE_API_KEY="$MICROSOFT_TRANSLATE_API_KEY" \
+    -e GOOGLE_TRANSLATE_API_KEY="$GOOGLE_TRANSLATE_API_KEY" \
+    -e WORDNIK_API_KEY='$WORDNIK_API_KEY' \
```

Comment:
why is only wordnik surrounded by single quotes?

Reply:
Ops, that's a mistake. Single quotes in bash will not replace the variable with it's value. I will fix it.